source/frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -78,6 +85,85 @@ export const DocumentDetailContainer: React.FunctionComponent<
     }
   };
 
+  const onUpdateDocument = async (updateRequest: ApiGen_Requests_DocumentUpdateRequest) => {
+    if (props.pimsDocument.id) {
+      const result = await updateDocument(props.pimsDocument.id, updateRequest);
+      result && props.onUpdateSuccess();
```

Comment:
safer to use if(exists(result))

Reply:
or checking by the contents of the result as well. result.data.id

Reply:
updated